packages/cf-component-progress/src/index.js

```diff
@@ -1,3 +1,6 @@
-import Progress from './Progress';
+import ProgressUnstyled from './Progress';
+import ProgressTheme from './ProgressTheme';
 
-export default Progress;
+const Progress = applyTheme(ProgressUnstyled, ProgressTheme);
+
+export { Progress as default, ProgressUnstyled, ProgressTheme };
```

Comment:
Well we do, both in the non-fela packages as well as in other ones such as [Toggle](https://github.com/cloudflare/cf-ui/blob/a49ebb5f87b31209b0beadada966fa8e8d3ed2bf/packages/cf-component-toggle/src/index.js).

Doing it this way means it doesn't break the "import API" so you don't need to update all the places where it's imported in the dashboard.

I'm fine if we want to not use the default keyword but there's very much a precedent to do it like this.